.github/workflows/rust.yml

```diff
@@ -39,12 +39,6 @@ jobs:
     - uses: actions/checkout@v2
       with:
         submodules: recursive
-    - name: Install LLVM and Clang # required for bindgen to work, see https://github.com/rust-lang/rust-bindgen/issues/1797
-      uses: KyleMayes/install-llvm-action@v1
-      if: matrix.os == 'windows-latest'
-      with:
-        version: "11.0"
-        directory: ${{ runner.temp }}/llvm
```

Comment:
This isn't needed anymore?

Reply:
We can ping @HKalbasi who added it in #218 to confirm but this was caught by the Github Actions linter.

It identifies that the `if: matrix.os == 'windows-latest'` will never be true so this step does not fire(this is verified by looking at the most recent run for this repository https://github.com/prove-rs/z3.rs/actions/runs/4537746078/jobs/7995914848). This condition tries to access this part of the context `matrix.os` that is never set in this specific job.

Additionally, this fix (afaik) is only needed when running on windows, this job uses `ubuntu-latest` which is why it this works despite the step never firing. In looking back at the original commit https://github.com/prove-rs/z3.rs/pull/218/commits/a4fb9dd195d8f1933464e677b13e33ddf8aef064, I'm unsure that this was ever needed in this specific job.

Reply:
You're correct! This isn't needed here since this doesn't run on Windows.